contracts/truefi/LoanToken.sol

```diff
@@ -394,6 +394,22 @@ contract LoanToken is ILoanToken, ERC20 {
         return _balance().add(redeemed);
     }
 
+    /**
+     * @dev Check whether an ongoing loan has been repaid in full
+     * @return true if and only if this loan has been repaid
+     */
+    function isRepaid() external override view returns (bool) {
+        return _isRepaid();
+    }
```

Comment:
Do we need both `isRepaid` and `_ isRepaid`? I'd say that making `isRepaid` public and using it internally should be good enough

Reply:
(See comment thread above)

I'd like to get rid of _isRepaid as well, but unfortunately I can't find a way to make it compile.
- If I make isRepaid public then the compiler complains that ILoanToken functions must be external, not public
- If I make isRepaid external then the compiler complains that we can't call it internally from close()

So it looks to me like we have to keep the internal/external split to satisfy the compiler. This also appears to be the workaround we're using for balance()/_balance() (lines 417/425) right now.

Reply:
You can set visibility to external in the interface and public in the implementation.

Reply:
Oh that worked!

Cool, thanks for the Solidity tip. Looks like interfaces here work differently from what I'd expected.